index.html

```diff
@@ -0,0 +1,120 @@
+            <h5 class="commentLine1">I have been off social media for 38 days and have came up with more life realizations and business/ life plans that I ever thought possible. I can also read for hours upon hours on end getting so dived up in what im reading that I forget the time. My mind simply has no reason to get distracted because there is nothing to distract my mind. This stuff works, where attention goes, energy flows! goodluck my friends</h5>
+         </div>
+    </div>
+    </div>
+        <div id="play_list">
+            <div class="thumb-image">
+                <a href="https://www.youtube.com/watch?v=_bJeKUosqoY&list=WL&index=115&t=631s">
+                    <img src="img/Capture.PNG" alt="Apple DOES Slow Down iPhones, iPhone X Jailbreak &amp; ">
+                </a>
+            </div> 
+            <div class="thumb-image">
+                <a href="https://youtu.be/YompsDlEdtc">
+                    <img src="img/Capture1.PNG" alt="Apple DOES Slow Down iPhones, iPhone X Jailbreak &amp; ">
+                </a>
+            </div><div class="thumb-image">
+                <a href="https://youtu.be/p3q5zWCw8J4">
+                    <img src="img/Capture2.PNG" alt="Apple DOES Slow Down iPhones, iPhone X Jailbreak &amp; ">
+                </a>
+            </div><div class="thumb-image">
+                <a href="https://youtu.be/T647CGsuOVU">
+                    <img src="img/Capture3.PNG" alt="Apple DOES Slow Down iPhones, iPhone X Jailbreak &amp; ">
+                </a>
+            </div><div class="thumb-image">
+                <a href="https://youtu.be/zlm1aajH6gY">
+                    <img src="img/Capture4.PNG" alt="Apple DOES Slow Down iPhones, iPhone X Jailbreak &amp; ">
+                </a>
+            </div><div class="thumb-image">
+                <a href="https://youtu.be/094y1Z2wpJg">
+                    <img src="img/Capture5.PNG" alt="Apple DOES Slow Down iPhones, iPhone X Jailbreak &amp; ">
+                </a>
+            </div><div class="thumb-image">
+                <a href="https://youtu.be/GNcFjFmqEc8">
+                    <img src="img/Capture6.PNG" alt="Apple DOES Slow Down iPhones, iPhone X Jailbreak &amp; ">
+                </a>
+            </div><div class="thumb-image">
+                <a href="https://youtu.be/OkmNXy7er84">
+                    <img src="img/Capture7.PNG" alt="Apple DOES Slow Down iPhones, iPhone X Jailbreak &amp; ">
+                </a>
+            </div><div class="thumb-image">
+                <a href="https://youtu.be/LnzuMJLZRdU">
+                    <img src="img/Capture8.PNG" alt="Apple DOES Slow Down iPhones, iPhone X Jailbreak &amp; ">
+                </a>
+            </div><div class="thumb-image">
+                <a href="https://youtu.be/VNGFep6rncY">
+                    <img src="img/Capture9.PNG" alt="Apple DOES Slow Down iPhones, iPhone X Jailbreak &amp; ">
+                </a>
+            </div>
+              
+        </div>
+    </div>
```

Comment:
please use <ul> tag instead of <div> in line 73 (with id="play_list") as <ul> is more semantic than <div> and also replace all the <div> children  inside by <li> tag